app/views/docs/configuration.phtml

```diff
@@ -22,10 +22,143 @@
+        </tr>
+        <tr>
+            <td><code>http://[HOSTNAME_OR_IP]/v1/account/sessions/oauth2/callback/github/console</code></td>
+        </tr>
+    </tbody>
+</table>
+
+<h3><a href="#post-install" id="post-install">Post Installation</a></h3>
+<p>
+    Check the <b>Redirect on update</b> box under the <b>Post installation</b> section.
+</p>
+
+
+<h3><a href="#github-webhooks" id="github-webhooks">Webhook</a></h3>
+<p>
+    GitHub will notify Appwrite about events like new commits using webhooks.
+    Under <b>Webhook</b>, you need to check the <b>Active</b> checkbox.
+    You also need to set the <b>Webhook URL</b> as <code>https://[HOSTNAME_OR_IP]/v1/vcs/github/events</code>.
+</p>
+
+<p>
+    If you're running Appwrite on <code>localhost</code>, GitHub can't send requests to Appwrite through webhooks and automatic deployments won't work.
+    You'll need host Appwrite on a server or use a proxy like <a href="https://ngrok.com/" target="_blank" rel="noopener">ngrok</a> to make Appwrite accessible to GitHub.
+</p>
+
+<h3><a href="#repository-permission" id="repository-permission">Repository Permissions</a></h3>
+<p>
+    Configure these permissions under the <b>Repository permission</b> dropdown.
+</p>
+<table class="full">
+    <thead>
+        <tr>
+            <td>Permission</td>
+            <td>Access</td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Administration</td>
+            <td>Read and write</td>
+        </tr>
+        <tr>
+            <td>Checks</td>
+            <td>Read and write.</td>
+        </tr>
+        <tr>
+            <td>Commit Statuses</td>
+            <td>Read and write</td>
+        </tr>
+        <tr>
+            <td>Contents</td>
+            <td>Read and write</td>
+        </tr>
+        <tr>
+            <td>Issues</td>
+            <td>Read and write</td>
+        </tr>
+        <tr>
+            <td>Metadata</td>
+            <td>Read-only</td>
+        </tr>
+        <tr>
+            <td>Pull requests</td>
+            <td>Read and write</td>
+        </tr>
+        <tr>
+            <td>Webhooks</td>
+            <td>Read and write</td>
+        </tr>
+    </tbody>
+</table>
+
+<h3><a href="#account-permission" id="account-permission">Account Permissions</a></h3>
+<p>
+    Configure these permissions under the <b>Account Permission</b> dropdown.
+</p>
+<table class="full">
+    <thead>
+        <tr>
+            <td>Permission</td>
+            <td>Access</td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Email address</td>
+            <td>Read-only</td>
+        </tr>
+    </tbody>
+</table>
+
+<h3><a href="#subscribe-events" id="subscribe-events">Subscribe to Events</a></h3>
+<p>
+    Select these under the <b>Subscribe to events</b> dropdown.
+</p>
+<table class="full">
+    <thead>
+        <tr>
+            <td>Event</td>
+            <td></td>
+            <td></td>
```

Comment:
are these cells left empty intentionally ?

Reply:
yes, for spacing

Reply:
Peep the screenshot

Reply:
<img width="503" alt="Screenshot 2023-08-31 at 4 13 51 PM" src="https://github.com/appwrite/docs/assets/29069505/b5eddf41-42d1-426f-bb87-6d49143b090e">